src/chigame/games/urls.py

```diff
@@ -10,4 +10,5 @@
     path("create/", views.GameCreateView.as_view(), name="game-create"),
     path("<int:pk>/edit", views.GameEditView.as_view(), name="game-edit"),
     path("<int:pk>/", views.GameDetailView.as_view(), name="game-detail"),
+    path("search/", views.search_results, name="game-search-results"),
```

Comment:
FYI if a user visits this path manually it'll raise an error:
<img width="975" alt="image" src="https://github.com/uchicago-cs/chigame/assets/86740254/c07f560a-bcb5-404b-b55c-cd7fe10d0f6e">

Reply:
https://letterboxd.com/search/

<img width="1009" alt="Screenshot 2023-11-13 at 2 09 51 PM" src="https://github.com/uchicago-cs/chigame/assets/69015620/d3e60529-89a8-4f95-a3c8-e0d04f3a92a6">